CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Contributions are accepted as pull requests.  Please observe our coding
+practices at https://github.com/City-of-Helsinki/bestpractice/ .
+Please make your pull requests short, elegant and only handling one
+issue at a time!
```

Comment:
>elegant and only handling one issue at a time!

You probably copy pasted this from somewhere, but I think that using "handling" here is not good English 🤔Sounds a bit like it could be Finglish.

>Please make your pull requests short, elegant and only hande one issue at a time!

Reply:
Yup, it's template mentioned in confluence. I made few improvements based on your suggestions.